packages/tocco-ui/src/Panel/PanelHeaderFooter.js

```diff
@@ -0,0 +1,68 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+
+import Button, {StyledButton} from '../Button'
+
+const StyledPanelHeaderFooter = styled.div`
+  && {
+    display: flex;
+
+    > div {
+      flex: 1 1 auto;
+    }
+
+    ${StyledButton} {
+      margin-left: auto;
+      align-self: flex-start;
+    }
+  }
+`
+
+class PanelHeaderFooter extends React.Component {
+  render() {
+    const {
+      // eslint-disable-next-line
+      children,
+      // eslint-disable-next-line
+      isOpen,
+      // eslint-disable-next-line
+      isToggleable,
+      showToggler,
+      // eslint-disable-next-line
+      toogleOpenState
+    } = this.props
+
+    return (
+      <StyledPanelHeaderFooter>
+        <div>
+          { React.Children.map(children, child => {
+            return React.cloneElement(child)
+          })}
+        </div>
+        { isToggleable
+          && showToggler
+          && <Button
+            icon={isOpen ? 'fa-minus' : 'fa-plus'}
+            onClick={() => toogleOpenState()}
```

Comment:
Simply reference toogleOpenState (toggleOpenState) instead of creating a new one

Reply:
see b4b74fe578a136ce8e8339413508afd6717b8f27